### ISSUE TITLE
The GFM Example doesn't seem to work.

### ISSUE THREAD

**Issue by missbach** [mojombo/github-flavored-markdown#31]
The GFM doesn't get parsed correctly at [the example page](https://github.com/mojombo/github-flavored-markdown/issues/1). (Linked from [the GFM help](http://github.github.com/github-flavored-markdown/)). Take a look at the comments section where some folks decided to render the markdown themselves.

**Comment by nyeates** [comment]
Agreed, I too saw this off the bat. I am wondering if it was done because there was no easy way to show the parsed output of that file directly?
